pkg/microservice/aslan/core/service/handler/service.go

```diff
@@ -130,11 +130,8 @@ func YamlValidator(c *gin.Context) {
 	}
 	resp := make([]*ValidatorResp, 0)
 	errMsgList := svcservice.YamlValidator(args)
-	if len(errMsgList) > 0 {
-		ctx.Logger.Errorf("svcservice.YamlValidator err : %v", errMsgList)
-		resp = append(resp, &ValidatorResp{
-			Message: "Invalid yaml format. The content must be a series of valid Kubernetes resources",
-		})
+	for _, errMsg := range errMsgList {
+		resp = append(resp, &ValidatorResp{Message: errMsg})
```

Comment:
why the above log and error message is missing?

Reply:
the func does not return err , the prvious change by loveforever is not right ,it eat the err detail